Traceback (most recent call last):
  File "/root/Steering-Diffusion/train-scripts/train_diff.py", line 69, in <module>
    device = f'cuda:{sys.argv[2]}'
IndexError: list index out of range